src/components/calculator.js

```diff
@@ -4,6 +4,7 @@ import NumbersPad from './numbers';
 import ComplexOperationsPad from './complexOperations';
```

Comment:
It would be nice to stick to the wireframe design in your calculator page as well.

This is your design:

![Screenshot 2023-07-12 at 6 13 52 AM](https://github.com/HFG43/math_magicians/assets/79506105/b4031120-583b-4eed-8431-554b5d3c8c25)

Please stick to this design:

![Screenshot 2023-07-12 at 6 15 03 AM](https://github.com/HFG43/math_magicians/assets/79506105/9aaf0adf-4f4e-455d-b811-e0670dde9905)

It is very important to follow design rules